go/oasis-node/cmd/debug/byzantine/tendermint.go

```diff
@@ -26,69 +23,12 @@ import (
 	"github.com/oasislabs/oasis-core/go/tendermint/service"
 )
 
-var _ epochtime.Backend = (*fakeTimeBackend)(nil)
```

Comment:
I'm not entirely sure why this was needed before and if it was indeed save to remove now with the refactor, but the byzantine tests seem to pass.

Reply:
this is a typecheck that fakeTimeBackend implements epochtime.Backend.

Reply:
> this is a typecheck that *fakeTimeBackend implements epochtime.Backend.

:grin: i removed the whole `fakeTimeBackend` and was wondering what's the need for it (not the line specifically)